src/test/java/com/ghilly/service/CityServiceRestTest.java

```diff
@@ -0,0 +1,76 @@
+package com.ghilly.service;
+
+import com.ghilly.exception.IdNotFoundException;
+import com.ghilly.exception.NameAlreadyExistsException;
+import com.ghilly.model.City;
+import com.ghilly.model.Country;
+import com.ghilly.repository.CityRepository;
+import com.ghilly.repository.CountryRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class CityServiceRestTest {
```

Comment:
Where test which checking the condition 'checkNameIsCorrect(cityName);'

Reply:
Done